### PR TITLE
TagEdit category resolver

### PR DESCRIPTION
**Pull request by peolic**
This just seems better than calling the API.

### CODE REVIEW

File: pkg/api/resolver_model_tag_edit.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+type tagEditResolver struct{ *Resolver }
+
+func (r *tagEditResolver) Category(ctx context.Context, obj *models.TagEdit) (*models.TagCategory, error) {
+	if obj.CategoryID == nil {
+		return nil, nil
+	}
+
+	qb := r.getRepoFactory(ctx).TagCategory()
+	categoryID, _ := uuid.FromString(*obj.CategoryID)
+	return qb.Find(categoryID)
```
Comment (peolic) [inline pkg/api/resolver_model_tag_edit.go]: If this PR is merged before #199, this needs to be modified.
```suggestion
	return qb.Find(*obj.CategoryID)
```

Comment (ghost) [inline pkg/api/resolver_model_tag_edit.go]: I recommend using a dataloader for this seeing as a scene could have a lot of tags, most of which will have the same categories. The tagResolver already uses one for categories.

Comment (ghost) [reply pkg/api/resolver_model_tag_edit.go]: Duh, no, this is a tagEdit, nevermind.

**Review by ghost — APPROVED** [review]
Yup, definitely better than using graphql.

**Pull request merged**